src/import-csv.js

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+import _ from 'lodash';
+import CouchBulkImporter from 'couchbulkimporter';
+import parse from 'csv-parse';
+import BulkBadger from 'bulkbadger';
+
+export function cli (file, url, ...args) {
+  if (!file) {
+    const msg = [
+      'Usage:',
+      '',
+      'nmo import-csv [file] [couchdb-url] [...<csv options> <pairs>]',
+    ].join('\n');
+    const err = new Error(msg);
+    err.type = 'EUSAGE';
+
+    throw err;
+  }
+
+  return importcsv.apply(importcsv, [file, url, convertArgsToKey(args)]);
+}
+
+export function convertArgsToKey (args) {
```

Comment:
you don't need that function any more if you configure nopt to parse your options.

the passed options should be a usual object literal